PopPUNK/network.py

```diff
@@ -675,15 +686,26 @@ def printClusters(G, rlist, outPrefix = "_clusters.csv", oldClusterFile = None,
         # Otherwise just number sequentially starting from 1
         else:
             cls_id = newClsIdx + 1
+            needs_unword = True
+
+        if write_unwords and needs_unword:
+            unword = next(unword_generator)
+        else:
+            unword = None
 
         for cluster_member in newCluster:
             clustering[cluster_member] = cls_id
+            if unword is not None:
+                cluster_unword[cluster_member] = unword
 
     # print clustering to file
     if printCSV:
         outFileName = outPrefix + "_clusters.csv"
         with open(outFileName, 'w') as cluster_file:
```

Comment:
Is `outPrefix` a suffix above? Sorry if I've lost the thread.

Reply:
It is the prefix, I lost the thread at some point (or we didn't update the docstring and changed the function, who knows). Correctly described now